content/03.prior-knowledge.representation-and-transfer.md

```diff
@@ -0,0 +1,55 @@
+#### Representation learning
+
+Representation learning, also called feature learning, is the process of learning features from raw data, where a feature is an individual variable or property.
+An algorithm or approach will construct features as part of training and, in the case of supervised feature learning, use those features to predict labels on input data. 
+Using an example from transcriptomics, an unsupervised method such as matrix factorization can be used to extract a low-dimensional representation of the gene-level data, learning features that are a combination of input genes' expression levels [@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3].
+Low-dimensional representations trained on a collection of transcriptomic data can then be used as input to supervised machine learning methods [@doi:10.1186/s12859-020-3427-8]. 
+Supervised neural networks used in medical imaging studies [@doi:10.1016/j.procs.2016.07.014] (reviewed in [@doi:10.1016/j.zemedi.2018.11.002] and [@doi:10.1098/rsif.2017.0387]), which are trained to predict labels or classes, are also an example of representation learning.
+
+Whether or not a learned representation or set of features is _useful_ depends on the task at hand.
+In a supervised setting, it may be sufficient for a feature to distinguish between classes, but if we hope to use a feature for biological discovery, we may prioritize intepretability.
+For example, learned features in the medical imaging domain may be a series of edges that constitute a blood vessel formation that discriminates between disease states or the learned features may not align with characteristics recognized as important by domain experts at all.
+From transcriptomics data, learned features could be coordinated sets of genes involved in a biological process that are descriptive in some way [@doi:10.1038/s41467-020-14666-6], but do not necessarily distinguish cases from controls in our study.
+
+In the rare disease domain, Dincer et al. leveraged publicly available acute myeloid leukemia (AML) gene expression data to improve the prediction of _in vitro_ drug responses [@doi:10.1101/278739]. 
```

Comment:
Really interesting, I never considered AML to be a rare disease, but I also didn't know what the incidence rate was. 1:25k is pretty rare!

Reply:
I generally use the Orphan Drug Act definition (any condition affecting fewer than 200,000 people in the US), which suggests to me that we should probably state that definition in the introduction + explicitly talk about how not everything we'll touch on will be applicable to every rare disease.

Reply:
Yeah, I use that definition too - in this case, it was more about ignorance on my end w/r/t AML incidence. Agree that we should use the ODA definition and add this to the manuscript!